tests/testthat/_snaps/release.md

```diff
@@ -99,3 +99,37 @@
       * [ ] Tweet
       * [ ] Add link to blog post in pkgdown news menu
 
+# RStudio-ness detection works
+
+    Code
+      writeLines(release_checklist("1.0.0", on_cran = TRUE))
+    Output
+      Prepare for release:
+      
+      * [ ] Check [current CRAN check results](https://cran.rstudio.org/web/checks/check_results_{TESTPKG}.html)
+      * [ ] [Polish NEWS](https://style.tidyverse.org/news.html#news-release)
+      * [ ] `urlchecker::url_check()`
+      * [ ] `devtools::check(remote = TRUE, manual = TRUE)`
+      * [ ] `devtools::check_win_devel()`
+      * [ ] `rhub::check_for_cran()`
+      * [ ] `revdepcheck::cloud_check()`
+      * [ ] Update `cran-comments.md`
+      * [ ] Draft blog post
+      * [ ] Ping Tracy Teal on Slack
```

Comment:
What are we supposed to ping Tracy about?

Reply:
for every release of every package, or only certain ones with certain more visible packages?

Reply:
For anything with a blog post